### DISCUSSION TITLE
Welcome to webERP Discussions!

### DISCUSSION THREAD

**Discussion by timschofield** [timschofield/webERP#141] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aulislap** [comment]
i have updated in my own environment (wamp) edi orders script, added missing segments, and i have added data into new chart, how i can update new sale order by using this data ?

**Comment by nutcracker53** [comment]
Hello , am Jaime and I am an ERP enthusiast . I have tried  weberp. ver 5.0.0. and I just read that ver 5.01 has been released. Can somebody please provide me with the link where I can download  weberp ver 5.01.  Thank you

**Reply by timschofield** [reply]
Hi Jaime, links to all webERP releases (including 5.0.1) going back 21 years can be found here - https://github.com/timschofield/webERP/releases

Tim
